### PR TITLE
refactor: About page

### PR DESCRIPTION
**Pull request by Nerixyz**
Refactors the about page. The main change is about the layout of contributors, which are now in a flow layout with headings. I've pretty much copied the example from Qt and adjusted it to fit the needs.
Contributors without an image now get a default image. Since I wasn't sure which one to use, I've included two:

| anon.png | anon2.png |
|---|---|
|  | firefox_2024-03-31_13-22-52 |

Additionally, I've made the following changes:

- Added word-wrap for the version label (it would overflow for me, creating a horizontal scrollbar)
- Added a missing space in the emoji datasource
- Renamed "Contributors" to "People" (since there's already a category of contributors)

### CODE REVIEW

File: resources/contributors.txt [hunk 1]
```diff
@@ -3,80 +3,89 @@
...
+@header Collaborators
+
+zneix | https://github.com/zneix | :/avatars/zneix.png
+Mm2PL | https://github.com/mm2pl | :/avatars/mm2pl.png
+YungLPR | https://github.com/leon-richardt |
+dnsge | https://github.com/dnsge |
+Felanbird | https://github.com/Felanbird |
+kornes | https://github.com/kornes |
+
+@header Contibutors
```
Comment (Wissididom) [inline resources/contributors.txt:26]: This spelling doesn't seem right to me (missing `r`)

Comment (Nerixyz) [reply resources/contributors.txt:26]: Oops, thank you for checking!

File: src/widgets/settingspages/AboutPage.cpp [hunk 2]
```diff
@@ -166,9 +168,22 @@ AboutPage::AboutPage()
...
+                    l->addLinebreak(8);
+                    continue;
+                }
+
                 QStringList contributorParts = line.split("|");
 
-                if (contributorParts.size() != 4)
+                if (contributorParts.size() != 3)
                 {
                     qCDebug(chatterinoWidget)
```
Comment (pajlada) [inline src/widgets/settingspages/AboutPage.cpp:188]: While you're here, can you change this to a warning?

File: src/widgets/settingspages/AboutPage.cpp [hunk 3]
```diff
@@ -178,39 +193,42 @@ AboutPage::AboutPage()
...
 
-                const auto addAvatar = [&avatarUrl, &contributorBox2] {
-                    if (!avatarUrl.isEmpty())
+                const auto addAvatar = [&] {
+                    auto *avatar = new QLabel();
+                    QPixmap avatarPixmap;
+                    if (avatarUrl.isEmpty())
+                    {
+                        // TODO: or anon.png
+                        avatarPixmap.load(":/avatars/anon2.png");
```
Comment (pajlada) [inline src/widgets/settingspages/AboutPage.cpp:210]: I like anon2

**Review by pajlada — APPROVED** [review]
Overall looks good

File: src/widgets/settingspages/AboutPage.cpp [hunk 4]
```diff
@@ -166,51 +168,67 @@ AboutPage::AboutPage()
...
 
+                if (line.startsWith(u"@header"))
+                {
+                    if (l->count() != 0)
+                    {
+                        l->addLinebreak(20);
+                    }
+                    auto *label = new QLabel(QStringLiteral("<h1>%1</h1>")
+                                                 .arg(line.mid(8).trimmed()));
+                    l->addWidget(label, FlowLayout::Flag::IgnoreColumnAlign);
```
[the remark this answers is not in the document]
Comment (Nerixyz) [reply src/widgets/settingspages/AboutPage.cpp:179]: oh oops

**Pull request merged**